docs/source/reference/config.rst

```diff
@@ -120,6 +120,9 @@ Available fields and semantics:
     # instances. SkyPilot will auto-create and reuse a service account (IAM
     # role) for AWS instances.
     #
+    # User Specified SERVICE_ACCOUNT (IAM role): The name of the remote identity
+    # to give the launched resouce.
```

Comment:
Not sure how we wanted to reference this specifically as an example. Open to suggestion here.